Documentation/ApiOverview/DependencyInjection/Index.rst

```diff
@@ -12,7 +12,7 @@
 Dependency injection
 ====================
 
-.. versionadded:: 10.0
+.. seealso::
```

Comment:
Do we really need the link to the changelog? The details of the changelog should be available on this page - so we do not need to link to it. I opt for removing the link.

Reply:
Ok, so lets just do pre-version-10 in this PR and version 10 Without backport?

Reply:
The patch is from my side okay, I only would also remove the link to the changelog as it has no benefit for the user as all information should be in here. If not, then I agree, it should be left in a follow-up the information has to be migrated into the docs here.

By now we say: Hey, we have super-duper docs about DI. But you can also view the same information in the changelog. But maybe, we missed something, so just in case we linked the changelog.